pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -130,10 +130,16 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 		glog.V(2).Infof("Successfully initialized cloud provider: %q from the config file: %q\n", server.CloudProvider, server.CloudConfigFile)
 	}
 
+	privilegedLoopbackOpenShiftClient, _, err := configapi.GetOpenShiftClient(options.MasterClients.OpenShiftLoopbackKubeConfig)
+	if err != nil {
+		return nil, err
+	}
+
 	// This is a placeholder to provide additional initialization
 	// objects to plugins
 	pluginInitializer := oadmission.PluginInitializer{
-		ProjectCache: projectCache,
+		OpenshiftClient: privilegedLoopbackOpenShiftClient,
```

Comment:
IIRC it didn't want to set the openshift client (`SetOpenShiftClient(client.Interface)`) without this.